src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -232,8 +249,24 @@ public void testPutLenient() throws Exception {
     @Test
     public void testPatch() throws Exception {
         // Create object
+        create();
+
+        final InputStream body = new ByteArrayInputStream(sparqlUpdate.getBytes());
+
+        // Update triples with sparql update
+        final FcrepoResponse response = client.patch(url)
+                .body(body)
+                .perform();
+
+        assertEquals(NO_CONTENT.getStatusCode(), response.getStatusCode());
+    }
+
+    @Test
+    public void testPatchEtagUpdated() throws Exception {
+        // Create object
         final FcrepoResponse createResp = create();
-        final String createdEtag = createResp.getHeaderValue(ETAG);
+        String createdEtag = createResp.getHeaderValue(ETAG);
+        createdEtag = createdEtag.substring(2, createdEtag.length());
```

Comment:
Actually, `EntityTag::toString` will contain the strength indicator prefix. nevermind.